src/components/Auth/Login/Login.react.js

```diff
@@ -159,13 +159,9 @@ class Login extends Component {
       }
       case 'password': {
         const password = event.target.value.trim();
-        let passwordErrorMessage = '';
-        if (!password || password.length < 6) {
-          passwordErrorMessage = 'Password should be atleast 6 characters';
```

Comment:
We should still show, `Password cannot be empty` message.

Reply:
sir its for the login panel.I have also attached the surge link .Please check it out

Reply:
Sir,if password is empty while login ,the login button will not enable.You may check

Reply:
@tanujvyas10 We still want to show. User may not be able to understand the reason for the disabled button. Having empty check and showing message is a standard practice.

Reply:
In that case, we can permanently enable the login button, and if user click **login** without entering the email or password or both,the user will get the notification below each of the inputs accordingly.Can we do that?